tests/unittests/tests-core/tests-core-ringbuffer.c

```diff
@@ -131,11 +131,33 @@ static void tests_core_ringbuffer_remove(void)
 
 }
 
+static void tests_core_ringbuffer_remove_underflow(void)
+{
+    char mem[3];
+    ringbuffer_t buf;
+    ringbuffer_init(&buf, mem, sizeof(mem));
+
+    ringbuffer_add_one(&buf, 0);
+    ringbuffer_add_one(&buf, 1);
+
+    ringbuffer_remove(&buf, 2);
+    TEST_ASSERT_EQUAL_INT(1, ringbuffer_empty(&buf));
+
+    ringbuffer_add_one(&buf, 0);
+    ringbuffer_add_one(&buf, 1);
+
+    ringbuffer_remove(&buf,1);
+
+    TEST_ASSERT_EQUAL_INT(1, ringbuffer_get_one(&buf));
+    TEST_ASSERT_EQUAL_INT(1, ringbuffer_empty(&buf));
```

Comment:
According to the doc, `ringbuffer_empty` returns 0 if not empty, so why testing against 1 ?

Reply:
`ringbuffer_get_one()` removes it from the buffer so after a call to `ringbuffer_remove` and `ringbuffer_get_one()` it should be empty.

Reply:
`ringbuffer_peek_one` only takes a look without removing

Reply:
Ok, I see :) Sorry for the noise :)